app/process_proposal.go

```diff
@@ -61,6 +61,9 @@ func (app *App) ProcessProposal(req abci.RequestProcessProposal) (resp abci.Resp
 			tx = blobTx.Tx
 		}
 
+		// todo: uncomment once we're sure this isn't consensus breaking
+		// sdkCtx = sdkCtx.WithTxBytes(tx)
+
```

Comment:
[optional] no point in including commented code that we don't plan on uncommenting.

```suggestion
```